discovery/discovery.go

```diff
@@ -161,3 +161,34 @@ func DiscoverEndpoints(app App, insecure bool) (eps *Endpoints, err error) {
 
 	return
 }
+
+func SimpleDiscoverEndpoints(app App, insecure bool) (eps *Endpoints, err error) {
+	if app.Labels["version"] == "" {
+		app.Labels["version"] = defaultVersion
+	}
+	tplVars := createTemplateVars(app)
+
+	de := &Endpoints{}
+
+	// Simple Discovery endpoints
+	schemes := []string{"https"}
+	if insecure {
+		schemes = append(schemes, "http")
+	}
+	for _, scheme := range schemes {
+		// Ignore not handled variables as {ext} isn't already rendered.
+		uri, _ := renderTemplate(fmt.Sprintf("%s://{name}-{version}-{os}-{arch}.{ext}", scheme), tplVars...)
+		sig, ok := renderTemplate(uri, "{ext}", "sig")
+		if ok {
+			de.Sig = append(de.Sig, sig)
+		}
+		aci, ok := renderTemplate(uri, "{ext}", "aci")
```

Comment:
Serving a sig over HTTP but an ACI over HTTPS (or vice versa) seems weird, should we really support this case?